.github/workflows/lint.yml

```diff
@@ -40,4 +40,4 @@ jobs:
         id: build
 
       - name: Run linter
-        run: yarn run prettier
```

Comment:
was this broken before?

Reply:
no just renamed it because its running both `forge fmt` and `prettier` so the name `lint` makes more sense than `prettier`